Update Configuration part for AWS security apps (#5625)
SumoLogic/sumologic-documentation: docs/integrations/amazon-aws/guardduty.md

</details>

## Collecting logs for the Amazon GuardDuty app

You can collect the Amazon GuardDuty logs using two methods: 

- [Method 1: Collecting Amazon GuardDuty logs using EventBridge](#method-1-collecting-amazon-guardduty-logs-using-eventbridge)
- (Optional) [Method 2: Collecting Amazon GuardDuty logs using Sumo Logic HTTP endpoint](#method-2-collecting-amazon-guardduty-logs-using-sumo-logic-http-endpoint)

### Method 1: Collecting Amazon GuardDuty logs using EventBridge

This method leverages AWS EventBridge to streamline the logging process by sending data directly to Sumo Logic via an HTTP endpoint. By eliminating intermediary services such as Lambda, it offers a more straightforward and cost-effective solution.

#### Step 1: Create an HTTP source in Sumo Logic

To create an HTTP source in Sumo Logic, see [HTTP Logs and Metrics Source](/docs/send-data/hosted-collectors/http-source/logs-metrics/#configure-an-httplogs-and-metrics-source).

#### Step 2: Configure EventBridge API destination

> 

Reviewer: ```suggestion
Follow the below steps to configure the EventBridge API destination:
```

> 1. Open your Amazon EventBridge Console.

Author: Added the link

1. In the navigation bar, click **API destinations**.
1. Click **Create destination**.
1. Enter a name for the API Destination.
1. Provide the HTTP Source URL from Sumo Logic.
1. Click **Create a new connection** to create a connection for the API destination.
  1. Provide a connection name.
  1. Keep the API Type as **Public**.
  1. Select **Basic (Username/Password)** in the **Authorization type**.

>   1. Add any random values for **Username** and **Password**.

Reviewer: ```suggestion
  1. Add any value of your choice for **Username** and **Password**.
```

> 1. Create the connection.

Reviewer: ```suggestion
```

#### Step 3: Create the EventBridge rule

> 

Reviewer: ```suggestion
Follow the below steps to create the EventBridge rule:
```

> 1. Click **Rules** and then click **Create rule**.

Reviewer: Can you mention where is this **Rules**?

1. Set the event source to **AWS services** and then select **Security Hub** as the AWS service.
1. Select **All Events** in Event Type.
1. Under **Select targets**, choose **EventBridge API destination**.
1. Select the API Destination created in Step 2.
1. Select **Create a new role for this specific resource** in the **Execution role**.
1. Click **Create** to activate the rule.

### Method 2: Collecting Amazon GuardDuty logs using Sumo Logic HTTP endpoint

This method uses an AWS Lambda function to process, store, and forward logs to Sumo Logic. While it offers a robust solution, it introduces additional AWS resources, such as Lambda, which can increase both cost and complexity.

- Amazon GuardDuty sends notifications based on CloudWatch events when new findings, or new occurrences of existing findings, are generated.
- A CloudWatch events rule enables CloudWatch to send events for the GuardDuty findings to the Sumo CloudWatchEventFunction Lambda function.

> - The Lambda function sends the events to an HTTP source on a Sumo hosted collector.

Reviewer: ```suggestion
- The Lambda function sends the events to an HTTP source on a Sumo Logic hosted collector.
```

</details>

## Collecting logs for the Amazon GuardDuty app

You can collect the Amazon GuardDuty logs using two methods: 

- [Method 1: Collecting Amazon GuardDuty logs using EventBridge](#method-1-collecting-amazon-guardduty-logs-using-eventbridge)

> - (Optional) [Method 2: Collecting Amazon GuardDuty logs using Sumo Logic HTTP endpoint](#method-2-collecting-amazon-guardduty-logs-using-sumo-logic-http-endpoint)

Reviewer 2: The opening sentence says there are two options, so I'd remove "Optional" here.

```suggestion
- [Method 2: Collecting Amazon GuardDuty logs using Sumo Logic HTTP endpoint](#method-2-collecting-amazon-guardduty-logs-using-sumo-logic-http-endpoint)
```

When you configure the HTTP Source, make a note of the HTTP Source Address URL. You will need it in the next step.  

> #### Step 2: Deploy Sumo GuardDuty events processor

Reviewer: ```suggestion
#### Step 2: Deploy Sumo Logic GuardDuty events processor
```
